Makefile

```diff
@@ -33,6 +33,7 @@ DOCKER_TARGETS := $(patsubst %, docker-%, $(TARGETS))
 .PHONY: $(TARGETS) $(DOCKER_TARGETS)
 
 GOOS := linux
+VERSION_PKG=$(MODULE)/internal/info
```

Comment:
Nit: Technically the use of this `VERSION_PKG` should also be part of this commit.